### ISSUE TITLE
This Project is Looking for New Leadership

### ISSUE THREAD

**Issue by chrisdick14** [Data4Democracy/election-transparency#54]
Scott, Rachel, and I have gotten a little busy and are looking for someone (or a group of people) to take over this project. Please let us know if you are interested.

**Comment by beatgeek** [comment]
Happy to help out in taking over and then pull in a group. Let me know what the expectations are.

**Comment by chrisdick14** [comment]
@beatgeek are you already on our slack channel?

**Comment by beatgeek** [comment]
Yes, already on your slack channel - just been out a bit. Want to hit me up there?
